sw/device/silicon_creator/rom/keys/README.md

```diff
@@ -99,3 +144,5 @@ c55ad015
 884c7d65
 9f86d081
 ```
+
+[ott]: #
```

Comment:
What is meant to go here?

Reply:
Sorry, that was meant to be a link to the doc of opentitantool but I could not find it (does it exist ?). Probably should remove that.